tests/spec/core/github-spec.js

```diff
@@ -120,14 +122,20 @@ describe("Core - Github", () => {
       return commitHistory;
     }
     it("generates a participate set of links (from string)", async () => {
-      const doc = await makeRSDoc({ ...l10n, ...stringOpt });
+      const doc = await makeRSDoc({
+        ...l10n,
+        ...stringOpt,
+      });
```

Comment:
Did prettier do it? lol

Reply:
Yes, but I think it was my fault too. I’ll revert it.